code/Extension/JSONTextExtension.php

```diff
@@ -44,28 +44,34 @@
 
 namespace PhpTek\JSONText\Extension;
 
-use SilverStripe\Control\Director;
-
-if (!Director::is_cli() && !class_exists('SilverStripe\CMS\Controllers\CMSPageEditController')) {
-    $msg = 'Please install the silverstripe/cms package in order to use this extension.';
-    throw new JSONTextException($msg);
-}
-
 use PhpTek\JSONText\Exception\JSONTextException;
 use PhpTek\JSONText\ORM\FieldType\JSONText;
-use SilverStripe\ORM\DataExtension;
-use SilverStripe\Control\Controller;
 use SilverStripe\CMS\Controllers\CMSPageEditController;
+use SilverStripe\Control\Controller;
+use SilverStripe\Control\Director;
+use SilverStripe\ORM\DataExtension;
 
 class JSONTextExtension extends DataExtension
 {
+
+    public function __construct()
+    {
+        // Helpful class applicability message
+        if (!Director::is_cli() && !class_exists('SilverStripe\CMS\Controllers\CMSPageEditController')) {
```

Comment:
Seeing as we've gone to the trouble of including it in a `use` block, let's just call `!class_exists(CMSPageEditController::class)`